packages/app/obojobo-document-engine/src/scss/_chunk-caption.scss

```diff
@@ -0,0 +1,16 @@
+@import '~styles/includes';
+
```

Comment:
My best guess is this was a little bit of something left from backing away from overly involved iframe/figure caption changes. I probably introduced a shared css file - but reverting some parts leaves this sitting out by itself kinda weirdly.

Perhaps I should pull it back into the materia chunk for now - since it makes no sense out here in a shared spot?